x-pack/plugins/cloud/server/config.ts

```diff
@@ -18,6 +18,16 @@ const apmConfigSchema = schema.object({
   ),
 });
 
+const fullStoryConfigSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: false }),
+  orgId: schema.conditional(
+    schema.siblingRef('enabled'),
+    true,
+    schema.string({ minLength: 1 }),
```

Comment:
Are we sure we want to set the min to 1?

Reply:
I wasn't sure how long these strings could be from FullStory. I doubt we'd ever have one that is only 1 character, but not sure where to put the limit (3, 4, 5?). I figured instead of trying to figure that out to just guard against the most likely misconfiguration which is an empty string.